tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -200,6 +202,8 @@ pub mod pallet {
 			parent_ss58_prefix: u16,
 		) -> DispatchResult {
 			T::ManageOrigin::ensure_origin(origin)?;
+			ensure!(Self::user_shielding_keys(&who).is_some(), Error::<T>::InvalidUserShieldingKey);
```

Comment:
Maybe we can save some error checking here: https://github.com/litentry/litentry-parachain/blob/32927fe47122c88e088c052f6ba7d3c08398d852/tee-worker/app-libs/stf/src/trusted_call.rs#L511-L514

e.g. just print the error logs instead.

Reply:
👌